examples/oft-solana/README.md

```diff
@@ -34,6 +36,41 @@ pnpm install
 pnpm test
 ```
 
+### Get Devnet SOL
+
+```bash
+solana airdrop 5 -u devnet
+```
+
+We recommend that you request 5 devnet SOL, which should be sufficient for this walkthrough. For the example here, we will be using Solana Devnet. If you are hitting rate limits after, you can also use the [official Solana faucet](https://faucet.solana.com/).
```

Comment:
Replace with "If you hit RPC rate limits after"

Reply:
Can you clarify what needs to be replaced ? The whole set of lines from 42 - 45 ? Or do you mean just the first 2 sentences ?

Reply:
or maybe just "If you are hitting rate limits after" yea ? I will assume this and push this commit now.